amnesia.py

```diff
@@ -9,6 +9,12 @@
 from tweepy.error import TweepError
 
 
+# By default, do not actually delete any tweet/like.
+# Just print out what will be deleted.
+# Set this to False to actually delete tweets/likes.
+DRY_RUN = True
```

Comment:
Sorry, I'm a little confused. I understand that we should take out ```DRY_RUN``` FROM ```amnesia.py```. However, I don't see an equivalent option in ```configs.py```. Do you mean that we should move the option there, or not provide the option at all?